unit_test/solver/matrix/test_equation_object.py

```diff
@@ -0,0 +1,64 @@
+#  Copyright (c) 2023. Deltares & TNO
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Test equation object."""
+import unittest
+
+import numpy as np
+
+from simulator_core.solver.matrix.equation_object import EquationObject
+
+
+class EquationObjectTest(unittest.TestCase):
+    """Test the equation object."""
+
+    def test_init(self):
+        """Test the init of the equation object."""
+        # arrange
+
+        # act
+        equation_object = EquationObject()  # act
```

Comment:
Kan deze "#act" niet worden verwijderd?

Reply:
Zou deze equation object ook niet naar de arrange moeten? Of doe je dat hier omdat je de init wil testen?

Reply:
# act staat er om linting tevreden te stellen en dat komt weer omdat er niet een echt arrange blok is omdat we de init method testen,